src/table/utils/accessibility-utils.ts

```diff
@@ -85,13 +85,18 @@ export const moveFocus = (
   return nextCell;
 };
 
+/**
+ * Finds the cell with tab stops and focuses that cell.
+ * If no cells has focus, it focuses the first body cell instead
+ */
 export const focusBodyFromHead = (
   rootElement: HTMLElement,
   setFocusedCellCoord: React.Dispatch<React.SetStateAction<[number, number]>>
 ) => {
-  const cell = findCellWithTabStop(rootElement);
-  const newCellCoord = getCellCoordFromCell(rootElement, cell);
-  cell.focus();
+  let cell = findCellWithTabStop(rootElement);
+  const newCellCoord = cell ? getCellCoordFromCell(rootElement, cell) : FIRST_BODY_CELL_COORD;
```

Comment:
I would suggest to change this function's name as well, 
`getCellCoords` should be fine, I think since we passing a cell into that function that should be obvious that we need coords of that cell and it requires a cell element

Reply:
also, I've checked that function, Not sure if it will work correctly in VT. have you tried that case? 

Seems like you cannot move tab stops more than last cell in the head 👇

https://user-images.githubusercontent.com/29652890/226286785-fecbf8cb-ce91-47ce-a5fc-f941edca8e55.mov